setup.py

```diff
@@ -25,7 +25,7 @@
 setup(
     name = "VALORANT rank yoinker",
     version = version,
-    description='vRY - Live Match Rank Viewer',
+    description='vRY - VALORANT rank yoinker',
     executables = [Executable("main.py", icon="./assets/Logo.ico", target_name="vry.exe")],
```

Comment:
```suggestion
    executables = [Executable("main.py", icon="./assets/Logo.ico", target_name="vRY.exe")],
```

Reply:
Are you sure you want to make the binary file case-sensitive?

Reply:
In that case, let me just revert the commit